onboarding/fibonacci.py

```diff
@@ -14,5 +14,17 @@ def fibonacci_term(n):
     @returns integer, nth term of the Fibonacci sequence
     '''
 
-    # insert code here
-    pass
+    # checking valid inputs
+    if type(n) != int or n < 1:
+      return None
```

Comment:
raise TypeError if type is not int. Please check the test cases and do appropriately.